gcs-action/.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
Did you need this file? I found that it was unnecessary with the new eslint configuration.

Reply:
Done